### PR TITLE
fix: migration generator indexes

### PR DESCRIPTION
**Pull request by nickschot**
A single index was created over `id`, `created_at` and `updated_at` in the default generator for migrations. This also gave problems with mysql (index too long).

### CODE REVIEW

**Comment by nickschot** [comment]
Will probably need to modify the migrations of the example applications too. But I'll do that tomorrow.

**Comment by nickschot** [comment]
Ugh, this is still wrong. Only way to do it is to set the indexes separately.

File: src/packages/cli/templates/model-migration.js [hunk 1]
```diff
@@ -58,11 +58,11 @@ export default (name: string, attrs: Array<string> | string): string => {
         ${body}
         table.timestamps();
 
-        table.index([
+        table.index(
           'id',
```
Comment (zacharygolba) [inline src/packages/cli/templates/model-migration.js]: I don't think we have to create an additional index for `id` as it will be the primary key.

Comment (nickschot) [inline src/packages/cli/templates/model-migration.js]: Indeed. That is covered. I noticed the need for separate indexes too in the comment before this. Knex has no API to add multiple indexes at once. I'll try to work on this a bit later tonight.

Comment (zacharygolba) [inline src/packages/cli/templates/model-migration.js]: 👍 Sounds good.

**Comment by zacharygolba** [comment]
👏  This looks good @nickschot. One last thing though. Can you update the migrations in the [examples](https://github.com/postlight/lux/tree/master/examples) as well as the [test-app](https://github.com/postlight/lux/tree/master/test/test-app)?

**Pull request merged**